.github/ISSUE_TEMPLATE/hotfix-release.md

```diff
@@ -0,0 +1,44 @@
+---
+name: k6 Hotfix release
+about: Bugs or security issues have been identified and they requires a patch release.
+title: 'k6 hotfix release v<major>.<minor>.<patch>'
+labels: ["release"]
+---
+
+**Release Date**:
+
+25th May 2024 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
+
+## Release Activities
+
+Note, the assumption is a minor branch release already exists from generated from the latest minor release. It has the name in the form `release-v<major>.<minor>.x`.
+
+#### Documentation
+
+- [ ] If required, open and merge pull-request from `main` applying the changes to the affected version.
```

Comment:
Strictly speaking, that's outside the scope of this PR but... what's the plan for [docs](https://github.com/grafana/k6-docs/tree/main/docs/sources)? 
Have a folder per minor? Like: `v1.0.x`,  `v1.1.x`, etc?

Reply:
Good question 👀 @heitortsergent do you maybe have insights on how you folks handle that in other projects?

Reply:
I don't have insights into how other projects handle minor versions, and we also handle versions differently than them.

@jdbaldry would you have any insight here? Is it just a matter of renaming/creating folders for minor versions?

Reply:
I'm not sure I understand, sorry. Are you not already maintaining a directory of documentation per minor version of k6?

Reply:
Oh yes, I misread it too, I'm sorry. I thought this was in case we had to release a patch version.

For other projects, they would just have a branch for each minor version, which is the equivalent of our folders in k6-docs. Is that right @jdbaldry?

Reply:
Yeah, most repositories have long-lived version branches for future patching and such, where the docs are maintained alongside the code. Since grafana/k6-docs is a docs-only repository, it's simpler to just have directories in the main branch instead.